projects/step-frontend/src/lib/modules/_common/components/login/login.component.scss

```diff
@@ -1,31 +1,113 @@
-mat-form-field {
+:host {
   width: 100%;
-
-  ::ng-deep .mat-form-field-flex {
-    background: none !important;
-  }
-
-  ::ng-deep input {
-    &:-webkit-autofill,
-    &:-webkit-autofill:hover,
-    &:-webkit-autofill:focus,
-    &:-webkit-autofill:active {
-      transition: background-color 5000s ease-in-out 0s;
-    }
-  }
+  display: flex;
 }
 
-:host {
-  width: 100%;
+.step-login-flex-child {
+  flex: 1;
   display: flex;
   flex-wrap: nowrap;
   justify-content: center;
   flex-direction: row;
   align-items: center;
+
+  .step-login-content {
+    width: 360px;
+    margin-top: -20vh;
+
+    .step-login-logo {
+      margin: 0 auto 30px;
+      width: 120px;
+
+      img {
+        max-width: 120px;
+        margin: 0 auto;
+      }
+    }
+
+    .step-login-heading-text {
+      text-align: center;
```

Comment:
By design the similar input will be used in other places.
@neogucky Should we adapt mat-inputs to this look, or we use our custom inputs in future?